virttest/utils_stress.py

```diff
@@ -210,6 +210,10 @@ def install_stressapptest(vm):
     :param vm: the vm to be installed with stressapptest
     """
     session = vm.wait_for_login(timeout=360)
+    if utils_package.package_install("stressapptest", session, timeout=300):
```

Comment:
Maybe this line code can be put before line 222, which is clustered code for stressapptest install

Reply:
> Maybe this line code can be put before line 222, which is clustered code for stressapptest install

I'm not sure if I understand you correctly. If you suggest this line be added before line 222 then as a result calling `install_stressapptest` would try to install git, gcc and other tools even if the stressapptest program is available. This doesn't make sense.